shell/platform/windows/flutter_window.cc

```diff
@@ -4,11 +4,15 @@
 
 #include "flutter/shell/platform/windows/flutter_window.h"
 
+#include <WinUser.h>
 #include <dwmapi.h>
 #include <chrono>
 #include <map>
```

Comment:
It looks like this code didn't quite follow the C++ style guide: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

It looks like C system headers should be separated from C++ headers, and, header sections should be sorted alphabetically.
 
```suggestion
#include <dwmapi.h>
#include <WinUser.h>

#include <chrono>
#include <map>
```

Reply:
They were originally sorted alphabetically, but the `linux_unopt` test failed with the required change listed being to place `WinUser.h` where it now is, so I am unsure if this is an appropriate change or not.

Reply:
Oh interesting, it may sort capitalized things first. I'd still separate the header groups though.

Reply:
The issue is that the C headers weren't separated from the C++ headers, and the linter is trying to ensure sorting within groups.

The order should be:

1. Header associated with the .cc file you're in, if you're in a .cc file. For tests, that's the header for the code under test.
2. C headers (for stdlib headers, prefer the C++ variant and put in the C++ group so `cmath` rather than `math.h` for example)
3. C++ headers
4. Flutter headers
5. Any `#ifdef`ed conditional headers.

Each group must be:
* separated from the others by a blank line.
* sorted within the group. My recollection is sorting is 'ASCII-betical' rather than alphabetical but the linter will catch this. 

For Windows, there are cases where it's necessary to place `<Windows.h>` or other headers near the top. In those cases, we typically include a comment as described in the Google style guide (link to relevant section in Loïc's comment).

Reply:
@yaakovschectman could you request my review next time (just so I get the notification). The comment above has content that I can definitely learn from.